src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV2.cs

```diff
@@ -37,11 +37,11 @@ public Task RunAsync(AgentTaskPluginExecutionContext context, CancellationToken
         // Properties set by tasks
         protected static class ArtifactEventProperties
         {
-            public static readonly string BuildType = "buildType";
+            public static readonly string PipelineType = "pipelineType";
             public static readonly string Project = "project";
```

Comment:
If we continue with the "source" naming for all of these "where the artifact comes from" inputs, I think we'd have:

* `sourceRun` (current or specific)
* `sourceProject`
* `sourcePipeline` (the ID/name of the pipeline aka definition)